### PR TITLE
fix en api docs of while, ifelse, switch

### PR DESCRIPTION
**Pull request by cryoco**
fix en api docs of while, ifelse, switch
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html (1)
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html (2)

### CODE REVIEW

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -643,29 +643,34 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
-            
-            i = fluid.layers.fill_constant(shape=[1], dtype='int64', value=0)
-            d0 = fluid.layers.data("d0", shape=[10], dtype='float32')
-            data_array = fluid.layers.array_write(x=d0, i=i)
-            array_len = fluid.layers.fill_constant(shape=[1],dtype='int64', value=3)
+            import numpy as np
+
+            i = fluid.layers.fill_constant(shape=[1], dtype='int64', value=0)           # 循环计数器
+
+            loop_len = fluid.layers.fill_constant(shape=[1],dtype='int64', value=10)    # 循环次数
```
Comment (NHZlX) [inline python/paddle/fluid/layers/control_flow.py]: 这里中文应改成英文

Comment (cryoco) [reply python/paddle/fluid/layers/control_flow.py]: done

**Review by Superjomn — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -643,29 +643,34 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
 class While(object):
     """
-    while loop control flow.
+    while loop control flow. Repeat while body until cond is False.
 
     Args:
-        cond(Variable): condition used to compare.
-        is_test(bool): A flag indicating whether execution is in test phase.
-        name(str): The name of this layer.
+        cond(Variable): A Tensor whose datatype is bool controlling whether to continue looping.
```
Comment (heavengate) [inline python/paddle/fluid/layers/control_flow.py]: datatype -> data type

Comment (cryoco) [reply python/paddle/fluid/layers/control_flow.py]: @heavengate Done. Thanks!

**Review by Superjomn — DISMISSED** [review]
LGTM

**Review by heavengate — DISMISSED** [review]

**Review by heavengate — APPROVED** [review]

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**